enterprise/server/remote_execution/uffd/uffd.go

```diff
@@ -272,6 +276,8 @@ func (h *Handler) handle(ctx context.Context, memoryStore *copy_on_write.COWStor
 			return nil
 		}
 
+		start := time.Now()
```

Comment:
Should we have this timer just wrap the `resolvePageFault` call (i.e. the UFFD copy)? since we have the operation metrics tagged with the "UFFD" file name I think it's redundant to have this count the COWStore operations (like you mentioned above)

Either way, I think this should at least go after the `readFaultingAddress` call because I think currently this measures the idle time while we're waiting for the guest to trigger a page fault?

Reply:
unix.Poll is what hangs when we're waiting for a page fault. 

But agree that there is some redundancy in the metric. Moved it to just cover the UFFD copy time